Makefile

```diff
@@ -121,6 +126,7 @@ mkinstalldirs :
 	mkdir -p "$(DESTDIR)$(docdir)/$(PACKAGE)"
 	mkdir -p "$(DESTDIR)$(libdir)/$(PACKAGE)"
 	mkdir -p "$(DESTDIR)$(sbindir)"
+	mkdir -p "$(DESTDIR)/usr/share/niroco.d"
```

Comment:
`$(datarootdir)` is provided for you.

Reply:
Retracted. `niroco` technically owns the location, so it's more appropriate to hard-code it.

Reply:
Ugh, I'm torn. This is trying to line up with the file layout of another package, such that the possibility of overriding `datarootdir` here could invalidate e.g. `$(DESTDIR)$(datarootdir)/niroco.d`.

I think the proper solution is another variable.